src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs

```diff
@@ -16,33 +17,60 @@ public class ResumeSQLRepository : IResumeSQLRepository
 
         public ResumeSQLRepository()
         {
+            IConfigurationRoot config = new ConfigurationBuilder()
+            .SetBasePath(Directory.GetCurrentDirectory())
+            .AddJsonFile("local.settings.json", optional: false, reloadOnChange: true)
+            .Build();
```

Comment:
no need to do this I think please sync with Lina

Reply:
@lina-ramirez-jalafund

Reply:
Solved